.github/pull_request_template.md

```diff
@@ -1,28 +1,47 @@
-## Brief Information
+<!-- 
+🚨NOTE🚨 Please complete the following tasks before submitting a pull request:
 
-This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):
+- [ ] Read the CONTRIBUTING doc. see: https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md
+- [ ] Sign the CLA. see: https://cla-assistant.io/hypertrons/hypertrons-crx
+-->
 
-- [ ] build
-- [ ] ci
-- [ ] docs
-- [ ] feat
-- [ ] fix
-- [ ] perf
-- [ ] refactor
-- [ ] test
+### Brief
 
-Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):
+This pull request is in the type of:
+<!-- 
+- `build`: Changes related to the build system, such as dependencies, packaging, and compiling.
+- `ci`: Changes related to continuous integration, such as configuration files and scripts.
+- `docs`: Changes related to documentation, such as README files and API documentation.
+- `feat`: New features or enhancements to existing features.
+- `fix`: Bug fixes or patches to existing features.
+- `perf`: Changes related to performance improvements.
+- `refactor`: Changes that improe code quality or structure without changing functionality.
+- `test`: Changes related to testing, such as adding or modifying unit tests. 
 
-- keyword #xxx
+Ref: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type
```

Comment:
Descriptions for types are really good except for a typo "improe" :)

I'm afraid  that `This pull request is in the type of:` is tend to be missed by newcomers. And another fact is that in practice a PR can belong to multiple types, but people don't like typing too much, so we resorted to checkboxes before.

Maybe we can figure out a better way.